src/api/users/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "user-microservice",
-  "version": "0.0.1",
+  "name": "users-microservice",
+  "version": "1.0.0",
```

Comment:
I like the 9-version bump here.  Go big!  In all seriousness, you've done enough review fixes to have earned this.  Great job staying the course with this PR and getting it banged into shape.

Reply:
Thanks for the excellent guidance along the way 😄, couldn't have done it alone.